agent/commands/run.go

```diff
@@ -59,8 +61,13 @@ func Run() {
 		l.Debugf("Loaded configuration: %+v", cfg)
 
 		cleanupTmp(cfg.Paths.TempDir, l)
+		if connectionUptimeService == nil {
+			logrus.Infof("Window check connection time is %.2f hour(s)", cfg.WindowConnectedTime.Hours())
+			connectionUptimeService = connectionuptime.NewService(cfg.WindowConnectedTime)
+			connectionUptimeService.RunCleanupGoroutine(ctx)
+		}
```

Comment:
let's implement window period updating logic.

Reply:
Implemented that